mmv1/products/compute/Disk.yaml

```diff
@@ -433,7 +433,6 @@ properties:
     description: |
       Whether this disk is using confidential compute mode.
       Note: Only supported on hyperdisk skus, disk_encryption_key is required when setting to true
```

Comment:
Just want to verify that this is correct, since it's not mentioned in the API docs? If not, could be good to fix in a follow-on PR.